eduardo/starwarsgem/lib/starwarsgem.rb

```diff
@@ -0,0 +1,38 @@
+require "starwarsgem/version"
+require "httparty"
+require 'json'
+
+module Starwarsgem
+  include HTTParty
+  base_uri 'https://swapi.co/api'
+
+  def self.find(type, id)
+    get("/#{type}/#{id}/.json")
+  end
+
+  def self.get_planet(planet_id = 3)
```

Comment:
In Ruby we don't use prefixes as `get`, `add`, `create`, `new`, `do`, `does`, `can`, etc...

Source: [Rubocop - Ruby Style Guide](https://github.com/rubocop-hq/ruby-style-guide#naming)

![image](https://user-images.githubusercontent.com/9273142/52285616-4ede4480-292c-11e9-9bc0-1cd8c0f4c074.png)